### PR TITLE
Models of epidemics

### PR DESCRIPTION
**Pull request by Liyubov**
First pull request on models of epidemics. 
I did not insert the code yet, but I put the links to it.

### CODE REVIEW

**Comment by Liyubov** [comment]
I can also add some figures to the modeling part. 
This is example of the image to demonstrate epidemics shift varying parameters of SIR model.
sirmodel_beta_mu_parameters

**Comment by rufuspollock** [comment]
@Liyubov graphs would be good. Also sounds like doing a short blog post would be great here! We can publish on datahub.io/blog 😄

File: coronavirus.md [hunk 1]
```diff
@@ -62,13 +62,19 @@ Coming soon ...
...
 ### France
 
 
 ## Models
-
-This section will list relevant modelling efforts.
-
-
+There are various types of models in epidemiology, which are used for prediction of prevalence (total number of infected) or the duration of an epidemic spreading processes. One of the most commonly used types is compartmental models, where population is divided into compartments of people with the same properties, e.g. people who are susceptible to virus, who are recovered etc.
+There are three main types of compartmental models in epidemiology:
```
Comment (rufuspollock) [inline coronavirus.md]: Line return 😄 See http://playbook.datopian.com/style-guide/markdown

File: coronavirus.md [hunk 2]
```diff
@@ -62,13 +62,19 @@ Coming soon ...
...
+There are three main types of compartmental models in epidemiology: 
+    Susceptible-Infected (SI), where each person can be just in two states, either susceptible (S) or infected (I); 
+     Susceptible-Infected-Recovered (SIS), where each person can be in one of two states, susceptible (S) or infected (I), but then can become susceptible again;
+     Susceptible-Infected-Recovered (SIR), where each person can be  in one of three states, susceptible (S), infected (I) or recovered (R).
+     
+In SIR model we study the number of people in each of the compartments, denoted by variable S, I and R correspondingly. SIR system can be  expressed by the set of ordinary differential equations proposed by [O. Kermack and Anderson Gray McKendrick](https://en.wikipedia.org/wiki/Compartmental_models_in_epidemiology). 
+The simulations of SIR model are shown in the python script [here](https://github.com/Liyubov/heterogeneous-dynamics-on-networks/blob/master/code_network_heterogen_models/spreading_SIR.py).
```
Comment (rufuspollock) [inline coronavirus.md]: Could you put this in an python notebook so people could see it and run it?

File: coronavirus.md [hunk 3]
```diff
@@ -62,13 +62,19 @@ Coming soon ...
...
+
+The main take-away message from the SIR model simulations is that we observe so-called peak of epidemic, when the number of infected people reaches its maximum. This moment of when size of epidemic outbreak is maximal depends on SIR model parameters: probability of epidemic transmission and recovery from the disease.
```
Comment (rufuspollock) [inline coronavirus.md]: English grammar here e.g. "This moment of when size of epidemic" (should be "The timing of this peak moment, when the size of the epidemic is maximal, depends on SIR model ..."

And could you say more about that ...

File: coronavirus.md [hunk 4]
```diff
@@ -75,7 +75,9 @@ There are three main types of compartmental models in epidemiology:
      Susceptible-Infected-Recovered (SIR), where each person can be  in one of three states, susceptible (S), infected (I) or recovered (R).
 In this model we study the number of people in each of those groups, denoted by variable S, I and R correspondingly. SIR system can be  expressed by the following set of ordinary differential equations: 
 
-$\frac{dS}{dt} = -\beta SI,$
```
Comment (rufuspollock) [inline coronavirus.md]: BTW we could enable mathjax which would allow you to add mathematical notation 😄

**Comment by Liyubov** [comment]
>  graphs would be good. Also sounds like doing a short blog post would be great here! We can publish on datahub.io/blog 😄

yes, i tried to insert graphs, like in the pull request reply, but i did not manage to insert the graph as you did for other graphs in the blog. 

maybe @anuveyatsu or anyone, 
you could help to insert the graph (attached below) into text about the model with different model parameters (beta and mu) and producing different curves of number of infected people? 
tnx!

sirmodel_beta_mu_parameters

**Pull request merged**